### ISSUE TITLE
Allow selection gui to persist

### ISSUE THREAD

**Issue by notthewave** [emersion/slurp#105]
I want to record part of my screen with wf-recorder. That is possible with slurp, however it's hard to perform actions within the selection if I don't have a constant indication of the recorded area, as slurp quits after selection.

https://user-images.githubusercontent.com/74874782/159025893-d4768402-8466-4d8f-aa64-5f5ba5a8db63.mp4

**Comment by dadav** [comment]
Good idea, but I think this should be done by the wf-recorder tool instead. My understanding of soup is, that it solves the task of finding some area/points. Indicating the area of recording doesn't quite fit in this task I think.
